dist/s_docs

```diff
@@ -160,6 +171,7 @@ build()
 {
 	# Build from scratch on demand.
 	[ "$1" -eq 0 ] || (cd .. && rm -rf docs && mkdir docs)
+	cp ../src/docs/js/* ../docs/
```

Comment:
Running `s_docs` in a fresh tree, I get an error:
    cp: directory ../docs does not exist

It looks like there isn't a top-level `docs/` directory in the repo, and in the preceding conditional `clean` is `0`, so we don't create the directory there.  I assume that the directory is normally created by `Doxygen`.

Reply:
I believe this is fixed, thanks.